backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/blobs/CassandraBlobMapper.java

```diff
@@ -57,16 +60,24 @@ public CassandraBlobMapper(Session session) {
     public CompletableFuture<UUID> write(byte[] data) {
         UUID uuid = UUIDs.timeBased();
         return FluentFutureStream.of(
```

Comment:
Why do you wrap in `FluentFutureSteam.of` just to call `.completableFuture()` on it. It seems useless